### ISSUE TITLE
Put up error message saying Process cannot be completed

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#100]
It doesn't have to say what the cause is, just that the server has crashed or whatever. So that the user doesn't just see the 25% progress bar for three hours thinking that the program is still working. 

Josh and Dan had ideas about how to implement this if you need to talk to them. 

Probably the message should say something like "The program has experienced a severe internal error. Please contact seed-support@lists.lbl.gov" or something like that.

**Comment by RDmitchell** [comment]
I have set this to Priority = 2, so do it after the Concatenation fix.

**Comment by RDmitchell** [comment]
Issues 32 and 33 are related, but they are more nuanced, ie, show what the actual problem is when there is an error, compared to this particular implementation, which is to just say that the server has died on data import so the user knows not to keep waiting for the program to finish the import.

**Comment by RDmitchell** [comment]
Here is a link to a file that will not import, to use for testing
https://drive.google.com/a/lbl.gov/file/d/0B3fTKpZ9Dx7LNXhZT05GM2FfQ0k/view?usp=sharing

**Comment by axelstudios** [comment]
Here's a screenshot of the initial changes when it catches a specific exception:
![seed - parse error](https://cloud.githubusercontent.com/assets/411466/7305746/61a2f618-e9bd-11e4-8178-444a3ed80333.png)

**Comment by rebrownlbnl** [comment]
This looks like a big improvement over what we currently have.
A couple thoughts:
- it looks like the error message is rendered in a pull-down list, or is that just an artifact of the way you've mocked it up? Seems like it should just be static text.
- We don't actually tell the user what they should do to resolve the error. In this case, I think they need to truncate the fields before import.
- Is the error logged anywhere?

**Comment by axelstudios** [comment]
I used the `pre-scrollable` class to prevent very-detailed error messages from extending beyond the bottom of the screen - Here's what it currently looks like with a long stacktrace:
![stacktrace](https://cloud.githubusercontent.com/assets/411466/7306568/7250c200-e9c3-11e4-8fa4-778294c35c08.png)

I wouldn't expect to ever give the user a stacktrace, but if you have detailed error output I think the scrolling helps.  I went ahead and made the class conditional, such that if the error message is less than 300 characters it won't show the scrollbar arrows.

I'm also working on making the error messages more descriptive, at least identifying which row caused the exception.  Currently, the error is only logged to the user's console and in Celery's stderr output.  We can definitely give Celery a more permanent logfile.
